native/packages/react-native-bpk-component-theming/src/BpkThemeProvider.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import { ThemeProvider } from 'theming';
```

Comment:
I reckon we wanna expose `withTheme` too - particularly since you are using it in your example above ;)

Reply:
Aye yeah - I put it in and then took it out, figured the PR would be the best place to make a decision. I'll re-add it.